src/index.ts

```diff
@@ -7,14 +7,18 @@ import {
   resolveMainMethodsCurrentFile,
 } from './debugserver';
 import { ISubstitutionVar } from './protocol';
+import { substituteFilterVariables } from './classFilter';
```

Comment:
This looks like an unused import. There is this and some other lint issues. Please run the fix task to clean them up.
```suggestion
```